<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZukeUI</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>zukebox.jpg</normaloff>zukebox.jpg</iconset>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>491</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="track_picture">
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>default.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="track_title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>9</y>
     <width>511</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QTextEdit" name="link_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText" stdset="0">
     <string>URL:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="message_edit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText" stdset="0">
     <string>Message:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>471</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="playorpause_button">
       <property name="text">
        <string>Play/Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="send_button">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QSlider" name="volume_slider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="seek_slider">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="track_sender">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>473</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="error_message">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>670</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>widget</zorder>
  <zorder>track_title</zorder>
  <zorder>volume_slider</zorder>
  <zorder>seek_slider</zorder>
  <zorder>track_sender</zorder>
  <zorder>error_message</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
